acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/jsonrpc/AbstractJsonRpcTest.java

```diff
@@ -95,7 +95,10 @@ public void test() throws IOException {
     final ObjectNode actualBody = JsonUtil.objectNodeFromString(response.body().string());
     final ObjectNode expectedBody =
         JsonUtil.objectNodeFromString(testCase.getResponse().toString());
-    assertThat(actualBody.toPrettyString()).isEqualTo(expectedBody.toPrettyString());
+    assertThat(actualBody)
+        .withFailMessage(
+            "%s\ndid not equal\n %s", actualBody.toPrettyString(), expectedBody.toPrettyString())
+        .isEqualTo(expectedBody);
```

Comment:
it was an ordering thing from prettyPrint?

Reply:
yeh, not sure why